test/unit/testreporter.js

```diff
@@ -1,5 +1,3 @@
-'use strict';
```

Comment:
The actual test files are being parsed by SystemJS, which will add a `'use strict';` directive to each module, but unless I'm mistaken *this* file runs as-is in the browser.

Hence it may be a good idea to keep this line, such that this file still runs in strict mode.